src/styles/main.scss

```
@@ -1,3 +1,198 @@
+html {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 24px;
+}
+
+h2 {
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  margin: 0;
+  padding-left: 0;
+  &__link {
```

```suggestion
  padding-left: 0;
 
  &__link {
```
Добавляй пустую линию между основным и вложенным тегом